playwright-tests/README.md

```diff
@@ -0,0 +1,37 @@
+## Playwright tests for the [supporter platform](https://support.theguardian.com/).
+
+Playwright tests written in Typescript have been added with the intention of replacing the existing [Scala Selenium tests](https://github.com/guardian/support-frontend/tree/main/support-frontend/test). The rational for this decision was primarily to allow us to write tests in the same language that the site is written in.
+
+As a first pass these tests will run alongside the existing Selenium tests while we migrate the test scenarios over to Playwright. When that work is complete we can remove the Selenium tests entirely.
+
+### Running the tests
+The tests have been setup to run via a github action. The action triggers on `pull_request` with the additional condition of the label `Seen-on-prod` being present (this label is added by prout once a deploy has finished and the changes have been confirmed in production).
+
+You can view the action [here](https://github.com/guardian/support-frontend/.github/workflows/playwright.yaml).
+
+Scripts to run the tests are present in the [package.json](https://github.com/guardian/support-frontend/playwright-tests/package.json) file. Allowing you to also run the tests locally. Three ways of running the tests have been setup:
+
+- `pw-browserstack-test` Runs the tests via BrowserStack Automate against the production site (https://support.theguardian.com/).
+- `pw-local-test` Runs the tests via the Playwright ui against a local running version of the site (https://support.thegulocal.com/).
+- `pw-prod-test` Runs the tests via the Playwright ui against the production site (https://support.theguardian.com/).
+
+It should also be possible to run the tests via BrowserStack Automate against a local running version of the site if you have local testing setup in your BrowserStack account and amend/create a new test configuration file.
+
+### Running the tests using BrowserStack
```

Comment:
Reading through this, I wonder where the value is of being able to run this locally with quite a complicated DevX? Maybe we could add it separately and maybe add some scripts to set it up?

Reply:
We've spoke about this and it would be good to support all workflows locally and in CI:
- Writing and running integration tests locally against playwright 
- For added confidence you should be able to run locally against browser stack
- Running automate in CI